auth/api/iam/s2s_vptoken.go

```diff
@@ -133,8 +137,8 @@ func (r Wrapper) createAccessToken(issuer did.DID, issueTime time.Time, presenta
 	}, nil
 }
 
-func (r Wrapper) accessTokenStore(issuer did.DID) storage.SessionStore {
-	return r.storageEngine.GetSessionDatabase().GetStore(accessTokenValidity, "s2s", issuer.String(), "accesstoken")
+func (r Wrapper) s2sAccessTokenStore() storage.SessionStore {
```

Comment:
This removes the tenant. Now an access token would be introspectable across tenant borders and thus usable on all tenants. It would depend on what the AAA does, but best is to return an error.

Reply:
The introspection EP currently doesn't know who is making the call, so the tenant is unknown. We need to discus what options we have here.

Reply:
That is indeed a fundamental change. Do we know for sure that the `aud` in the access token request is transferred to the `iss`/`sub`? This makes the access token introspection determine the tenant. That would be OK, but best to check A-Z if we can't leak anything across tenant boundaries.

Reply:
The presentation definition should extract tenant (did) from the vp_token to the PDPMap as discussed so it can be validated at the AAA.